### PR TITLE
Open data support

### PR DESCRIPTION
**Pull request by EthanMarx**
Adds support for querying open data

Set `channel` variable to `"OPEN"` and `state_flag` to `"DATA"` (not the most intuitive but made things easy).

### CODE REVIEW

**Comment by EthanMarx** [comment]
There are still a few minor issues: See [this](https://github.com/gwpy/gwpy/pull/1664) PR I opened with gwpy

File: projects/sandbox/pyproject.toml [hunk 1]
```diff
@@ -194,7 +194,7 @@ stop = "${base.test_stop}"
 psd_length = "${base.inference_psd_length}"
 sample_rate = "${base.sample_rate}"
 ifos = "${base.ifos}"
-state_flag = "${base.state_flag}"
+#state_flag = "${base.state_flag}"
```
Comment (wbenoit26) [inline projects/sandbox/pyproject.toml:197]: I think this should be uncommented

**Comment by wbenoit26** [comment]
It looks like you maybe originally used `state_flag = None` to indicate that we wanted open data, but then switched to using `state_flag = "DATA"`, because the default `None` won't currently produce a sensible state flag.

File: projects/sandbox/datagen/datagen/scripts/background.py [hunk 2]
```diff
@@ -235,11 +237,11 @@ def deploy(
     ifos: List[str],
     sample_rate: float,
     channel: str,
-    state_flag: str,
     datadir: Path,
     logdir: Path,
     accounting_group: str,
     accounting_group_user: str,
+    state_flag: Optional[str] = None,
```
Comment (wbenoit26) [inline projects/sandbox/datagen/datagen/scripts/background.py:244]: I'll suggest also changing this to be `"DATA"`, as well as the argument in `deploy`

File: projects/sandbox/datagen/datagen/scripts/background.py [hunk 3]
```diff
@@ -274,7 +276,9 @@ def deploy(
         state_flag:
             Identifier for which segments to use. Descriptions of flags
             and there usage can be found here:
-            https://wiki.ligo.org/DetChar/DataQuality/AligoFlags
+            https://wiki.ligo.org/DetChar/DataQuality/AligoFlags. If None,
```
Comment (wbenoit26) [inline projects/sandbox/datagen/datagen/scripts/background.py:279]: This comment should be updated as well

File: projects/sandbox/datagen/datagen/scripts/timeslide_waveforms.py [hunk 4]
```diff
@@ -269,6 +269,7 @@ def deploy(
     logdir: Path,
     accounting_group_user: str,
     accounting_group: str,
+    state_flag: Optional[str] = None,
```
Comment (wbenoit26) [inline projects/sandbox/datagen/datagen/scripts/timeslide_waveforms.py:272]: I think this and the docstring description/position should be updated as well.

Comment (wbenoit26) [reply projects/sandbox/datagen/datagen/scripts/timeslide_waveforms.py:272]: Oh, and could you move the docstring description in the other function as well. Sorry, didn't notice that the argument had moved.

**Comment by EthanMarx** [comment]
@alecgunny @wbenoit26 this is ready to go. Ran end to end through background and timeslide waveform generation

**Review by wbenoit26 — APPROVED** [review]

**Comment by wbenoit26** [comment]
Other than the conflicts, this looks good to me.

**Pull request merged**